### PR TITLE
Hotfix: sort output tag mismatch reverting breaking change

### PR DESCRIPTION
**Pull request by sbe-arg**
Okay the problem here was output tag as replaced with $new tag and that is breaking change so added new output old_tag instead and left the tag output as was before untill v2 

https://github.com/anothrNick/github-tag-action/issues/200

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -9,12 +9,13 @@ on:
 
 jobs:
   build:
+    if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true
```
Comment (sammcj) [inline .github/workflows/main.yml:12]: Why not just change the on pull-request to only be on: push to branch master?

Comment (sbe-arg) [reply .github/workflows/main.yml:12]: Because will make a tag with direct commits to master if a PR is bypassed

Comment (scott-doyland-burrows) [reply .github/workflows/main.yml:12]: Then the ```ref: ${{ github.ref_name }}``` can also be removed as well.

Comment (sammcj) [reply .github/workflows/main.yml:12]: There's branch protection so you can't push directly to master.

**Review by sammcj — APPROVED** [review]

**Pull request merged**